packages/components/tooltip/src/tooltip.tsx

```diff
@@ -75,7 +76,7 @@ const Tooltip = forwardRef<"div", TooltipProps>((props, ref) => {
   );
 
   return (
-    <>
+    <RemoveScroll enabled={shouldBlockScroll && isOpen} removeScrollBar={false}>
       {trigger}
```

Comment:
@wingkwong

This causes the trigger to be wrapped in a `div`. RemoveScroll should be wrapped in content!?

Why not use the `usePreventScroll` Hook?

Reply:
@chirokas That's a good point. I forgot you've made that PR already. Can you also apply the changes in your PR so that I could close this one?

Reply:
OK, I'll do it tomorrow.